src/UI/Internal/Palette.elm

```diff
@@ -0,0 +1,110 @@
+module UI.Internal.Palette exposing
+    ( Color
+    , ToneColors
+    , danger
+    , gray
+    , primary
+    , success
+    , textDisbledWithGrayLightest
+    , textWithBg
+    , textWithGrayLighter
+    , textWithGrayLightest
+    , warning
+    )
+
+import Element exposing (rgb255)
+
+
+type alias Color =
+    Element.Color
+
+
+
+-- Tones
+
+
+type alias ToneColors =
+    { darkest : Color
+    , middle : Color
+    , light : Color
+    , lighter : Color
+    , lightest : Color
+    }
+
+
+gray : ToneColors
+gray =
+    { darkest = rgb255 14 20 32 -- #0E1420
```

Comment:
I liked the idea of adding the hex color :D can you add in all the colors? I know it's boring but I think it's a good idea